app/controllers/users.js

```diff
@@ -16,4 +19,26 @@ const save = (request, response) =>
       response.status(400).json(error.original.detail);
     });
 
-module.exports = { save };
+const login = (request, response) =>
+  users
+    .findOne({ where: { email: request.body.email } })
+    .then(user => {
+      if (user) {
+        bcrypt.compare(request.body.password, user.password, (err, res) => {
```

Comment:
bcrypt does not have a promisified method? if so, use it. If not, you can make a helper so you dont use callbacks